Sources/Views/MessageLabel.swift

```diff
@@ -334,6 +366,24 @@ open class MessageLabel: UILabel {
         return results
     }
 
+    /**
+     Take a custom detector and apply the matching to the text
+     
+     - Parameters: detector: `DetectorType` that you want to execute
+     - Parameters: text: where we apply the regular expression
+     - Parameters: range: where we apply the regular expression
+     
+     - Returns: an array of `NSTextCheckingResult` that contains all maching for this detector or nil if it fails
+     */
+    private func parseForMatches(with detector: DetectorType, in text: NSAttributedString, for range: NSRange) -> [NSTextCheckingResult] {
```

Comment:
Docs are nice for me to understand the code, but fwiw we don't _need_ to document non-public APIs. I can't complain though 🥇

Reply:
Ok as you want 👍